packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/translations/Messages_fr_FR.json

```
@@ -0,0 +1,9 @@
+{
+  "pci_databases_service_integration_tab_title": "Intégration de services",
```

I think it's better:
```suggestion
  "pci_databases_service_integration_tab_title": "Intégration de service Kafka",
```

---

same: I cannot change the text as it has been already validated by content team.